### ISSUE TITLE
CRAN release?

### ISSUE THREAD

**Issue by jameslamb** [uptake/pkgnet#288]
@bburns632 @jayqi  it's been a bit over a year since we did a release to CRAN. Want to do one?

I found myself using `{pkgnet}` this week to prep for a talk, and I really wanted the features from https://github.com/uptake/pkgnet/pull/243, which I realize now never made it to CRAN.

If you're open to spending a little time on `{pkgnet}`, I'd also be happy to submit PRs for the warnings on https://cran.r-project.org/web/checks/check_results_pkgnet.html

```text
Version: 0.4.1
Check: LazyData
Result: NOTE
     'LazyData' is specified without a 'data' directory
Flavors: r-devel-linux-x86_64-debian-clang, r-devel-linux-x86_64-debian-gcc, r-devel-linux-x86_64-fedora-clang, r-devel-linux-x86_64-fedora-gcc, r-devel-windows-ix86+x86_64, r-devel-windows-x86_64-gcc10-UCRT, r-patched-linux-x86_64, r-patched-solaris-x86

Version: 0.4.1
Check: Rd cross-references
Result: NOTE
    Undeclared packages ‘pkgdown’, ‘htmlwidgets’ in Rd xrefs
Flavor: r-devel-linux-x86_64-fedora-clang
```

**Comment by bburns632** [comment]
Has it been that long!? Yes, let's do one to incorporate #243 and keep up
with R changes / CRAN checks.

**Comment by jameslamb** [comment]
ok cool, thanks! No rush at all, I'll try to get to those warnings when I can (or anyone finding this issue who wants to contribute is welcome to!).

**Comment by bburns632** [comment]
LazyData message resolved in #289
